config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -8,6 +8,10 @@ image:
   pullPolicy: IfNotPresent
   pullSecrets: []
 
+# -- Custom labels to add into metadata
+customLabels: {}
```

Comment:
can you add this to the helm readme under the general config section with a description?